common/db_store/db.py

```diff
@@ -90,26 +90,66 @@ def get_data(self, data_store_name: str, **kwargs):
 
         Args:
             data_store_name (str): Name of data store.
-            **kwargs: queryparams (json/bson): JSON / BSON string.
+
+        Keyword Args:
+            **kwargs: Arbitrary keyword arguments.
+            queryparams (dict): dict following pymongo convention.
+            sort_query (List of tuples): (field_to_sort, sort_direction)
+                direction can be ASC/ASCENDING or DSC/DESCENDING.
+            limit (int): Limit for no. of documents returned by query.
+            skip (int): To skip the first x records of the returned results.
+            count(bool): True / False. To get the total count of documents
+                irrespective of limit & skip field.
 
         Raises:
             DBError: Unable to fetch data.
 
         Returns:
-            list: List of records.
+            tuple: (List of records, Total count of documents)
```

Comment:
I think it better to return the first number then a List of records.
call is your.

Reply:
The objective of get_data method is to get list of records, also by-default it return count as None if you haven't set count=True. That's why I kept it as second parameter.